### PR TITLE
fix: added version to runtime picker

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1257

### CODE REVIEW

**Comment by fabric8cd** [comment]
This PR was deployed **with mocked data** in the following URLs for your appreciation:
- Frontend: https://fabric8-launcher-frontend-pr-1282-app.surge.sh
- Welcome App: https://fabric8-launcher-frontend-pr-1282-welcome-app.surge.sh
- Storybook: https://fabric8-launcher-frontend-pr-1282-storybook.surge.sh

File: packages/launcher-component/src/pickers/runtime-picker.tsx [hunk 1]
```diff
@@ -2,44 +2,43 @@ import * as React from 'react';
...
 };
 
 export const RuntimePicker: Picker<RuntimePickerProps, RuntimePickerValue> = {
-  checkCompletion: value => !!value.id,
+  checkCompletion: value => !!value.runtimeId,
```
Comment (ia3andy) [inline packages/launcher-component/src/pickers/runtime-picker.tsx]: @edewit you should check `versionId` too right?

File: packages/launcher-component/src/flows/launcher-client-adapters.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export function buildDownloadNewAppPayload(app: NewApp) {
     parts.push({
       category: 'backend',
       shared: {
-        runtime: {name: app.backend.runtimePickerValue!.id, version: 'community'}
+        runtime: {name: app.backend.runtimePickerValue!.runtimeId, version: 'community'}
```
Comment (ia3andy) [inline packages/launcher-component/src/flows/launcher-client-adapters.ts]: I guess you have to send the versionId here and at line 21?

File: packages/launcher-component/src/pickers/runtime-picker.tsx (hunk 1 above)
Comment (edewit) [reply packages/launcher-component/src/pickers/runtime-picker.tsx]: but the versionId is sent to the first value

File: packages/launcher-component/src/flows/launcher-client-adapters.ts (hunk 2 above)
Comment (edewit) [reply packages/launcher-component/src/flows/launcher-client-adapters.ts]: good catch

**Pull request merged**